packages/i18n/src/locales/pt-BR.i18n.json

```diff
@@ -4867,6 +4870,9 @@
   "You_have_not_verified_your_email": "Você ainda não verificou o seu e-mail.",
   "You_have_successfully_unsubscribed": "A partir de agora você não está mais cadastrado em nossa lista de e-mails.",
   "You_must_join_to_view_messages_in_this_channel": "Você deve participar para ver mensagens neste canal",
+  "You_mentioned___mentions__but_theyre_not_in_this_room": "Você mencionou {{mentions}}, mas eles não estão nesta sala.",
```

Comment:
Can you use [plurals](https://www.i18next.com/translation-function/plurals) here?

Reply:
<img width="518" alt="image" src="https://github.com/user-attachments/assets/175e30e6-81f1-4af0-8d60-a3ccf6c073b7">
Here you are mentioning a single user, but the rest of the sentence is for more than a single one